spec/models/views/summary_spec.rb

```diff
@@ -162,7 +162,7 @@
       let(:online_application) { build(:online_application, refund: true, date_fee_paid: '01/02/2016') }
 
       it 'returns Yes with date paid' do
-        expect(subject).to eql('Yes, on 01/02/2016')
+        expect(subject).to eql("Yes, on #{online_application.date_fee_paid}")
```

Comment:
@lambley we need to test the specific value here. not just hope that the value is there by pulling it from the model. Which is probably replicating what the "normal" code is doing.